components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -86,6 +86,7 @@ class RhsRootPost extends React.PureComponent {
         };
 
         this.postHeaderRef = React.createRef();
+        this.dotMenu = React.createRef();
```

Comment:
minor change, can we rename to dotMenuRef for consistency with other PRs
```suggestion
        this.dotMenuRef = React.createRef();
```

Reply:
the change is done